rotkehlchen/tasks/utils.py

```diff
@@ -22,3 +25,18 @@ def should_check_data_updates(database: 'DBHandler') -> bool:
 
     last_update_ts = deserialize_timestamp(timestamp_in_db)
     return ts_now() - last_update_ts >= DATA_UPDATES_REFRESH
+
+
+def should_detect_evm_accounts(database: 'DBHandler') -> bool:
```

Comment:
nope as I said to both you and @yabirgb you should combine those two functions. They are identical. The only thing missing is the arguments: `key_name`, `refresh_period`